src/hooks/transactions/useTransactionType.ts

```diff
@@ -29,9 +29,9 @@ interface TransactionFunction {
 }
 
 interface Props {
-  input: string;
+  input: string | null;
   type: string;
-  weightInput?: string;
+  weightInput?: string | null;
```

Comment:
It was nicer if the types were non-nullable. We can solve that by having the `useDebouncedState` always require the initial state or moving to  generic parameters.